<launch>
    <group ns="uav1">
        <node pkg="yolov5_detect" type="yolov5_detect" name="yolov5_detect_node" output="screen">

            <param name="engine_name" value = "$(find yolov5_detect)/weights/0410best_s.engine"/>	
            <param name="video_name" value = "$(find yolov5_detect)/detect/test_vedio/DJI_0654.MP4"/> 
            <!-- <param name="video_name" value = "rtsp://admin:admin@192.168.42.108:554/cam/realmonitor?channel=1&amp;subtype=0"/>  -->
            <param name="img_topic" value = "prometheus/sensor/monocular_down/image_raw"/> 
            <param name="uav_num" value = "1"/> 
            <param name="video_out_path" value = "$(find yolov5_detect)/detect/"/> 

        </node>
    </group>

    <group ns="uav2">
        <node pkg="yolov5_detect" type="yolov5_detect" name="yolov5_detect_node" output="screen">

            <param name="engine_name" value = "$(find yolov5_detect)/weights/0410best_s.engine"/>	
            <param name="video_name" value = "$(find yolov5_detect)/detect/test_vedio/DJI_0654.MP4"/> 
            <!-- <param name="video_name" value = "rtsp://admin:admin@192.168.42.108:554/cam/realmonitor?channel=1&amp;subtype=0"/>  -->
            <param name="img_topic" value = "prometheus/sensor/monocular_down/image_raw"/> 
            <param name="uav_num" value = "1"/> 
            <param name="video_out_path" value = "$(find yolov5_detect)/detect/"/> 

        </node>
    </group>
</launch>
